pulseapi/entries/test_views.py

```diff
@@ -0,0 +1,67 @@
+from django.core.urlresolvers import reverse
+from django.test import TestCase
+
+from pulseapi.entries.serializers import EntrySerializer
+from pulseapi.entries.test_models import EntryFactory
+
+
+class TestEntryView(TestCase):
+    def setUp(self):
+        self.entries = [EntryFactory() for i in range(2)]
+        for entry in self.entries:
+            entry.save()
+
+    def test_get_single_entry_data(self):
+        """
+        Check if we can get a single entry by its `id`
+        """
+
+        id = self.entries[0].id
+        response = self.client.get(reverse('entry', kwargs={'pk': id}))
+        self.assertEqual(response.status_code, 200)
+
+    def test_post_minimum_entry(self):
+        #Entry with minimum amount of content (title, description?)
```

Comment:
do we want negative tests, too, so that there's no way to regress to a model where title+description are no longer necessary?